src/promql.grammar

```diff
@@ -256,13 +256,9 @@ SignedNumber{
   whitespace { std.whitespace+ }
   LineComment { "#" ![\n]* }
 
-  NumberLiteral {
-    ("+" | "-")? (
+  digit {
```

Comment:
A digit is a single character, like `1` or `2`. Maybe still call it `number`?

Reply:
yup it works.